src/modules/notifications/notifications.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable, Logger } from "@nestjs/common";
+import { FirebaseService } from "../firebase/firebase.service";
+import { HandleException } from "src/common/exceptions/handler/handle.exception";
+import { UsersService } from "../users/users.service";
+import { NotificationDTO } from "../firebase/models/firebase.request.dto";
+import { title } from "process";
+import { UpdateAppRequestDTO } from "./models/update.app.request.dto";
+import { stringConstants } from "src/utils/string.constant";
+
+@Injectable()
+export class NotificationsService {
+  constructor(
+    private readonly firebaseService: FirebaseService,
+    private readonly userService: UsersService,
+  ) {} 
+Ò
+
+
+  appUpdateNotification = async (body: UpdateAppRequestDTO) => {
+    try {
+        Logger.log("Body ->", JSON.stringify(body));
```

Comment:
do you need this logger?

Reply:
I don't need it, in fact I don't remember adding it, maybe I forgot